app/assets/stylesheets/global/html_forms/elmo_forms/_hints.scss

```diff
@@ -17,6 +17,14 @@
       padding: 0 3px;
 
       vertical-align: middle;
+
+      cursor: pointer;
```

Comment:
Can you remove the blank lines above and move the comment to the right of the `3px;` or otherwise tidy.